src/app/saga.ts

```diff
@@ -339,6 +339,14 @@ export function* handleDeepLink(action: OpenDeepLink) {
   const rawParams = parse(deepLink)
   if (rawParams.path) {
     const pathParts = rawParams.path.split('/')
+    ValoraAnalytics.track(AppEvents.handle_deeplink, {
+      pathStartsWith: pathParts[1].split('?')[0],
+      fullPath:
+        pathParts[1] === 'jumpstart'
+          ? 'redacted-due-to-presense-of-private-key'
```

Comment:
don't want to cause a privacy issue

Reply:
nit: maybe enclose these in `[]` so it doesn't look similar to other full paths

Reply:
Wow, didn't realize we had a deeplink that passes in a private key! :exploding_head: 

This does feel a bit brittle though wrt avoiding privacy issues. If a deeplink is added that contains sensitive information in the path, this will need to be manually updated, which we could easily forget to do. Probably not a big deal, but thought I'd call it out - maybe worth a comment in code?

Reply:
great catch @jh2oman . maybe instead of including query params by default, we could allowlist them, to err on the side of caution here